extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -705,6 +710,7 @@ public void handle(AsyncResult<Void> event) {
                                     }
                                 });
                             } catch (Exception e) {
+                                LOGGER.warn("Failed to undeploy deployment.");
                                 e.printStackTrace();
```

Comment:
Instead of keeping the `e.printStackTrace();`, include `e` as the cause in the `warn()` call.